Makefile

```diff
@@ -270,6 +271,9 @@ format: $(wildcard src/toil/cwl/*.py)
 	black $^ contrib/mypy-stubs
 
 mypy:
+	mypy --ignore-missing-imports --no-strict-optional \
+		--warn-redundant-casts --warn-unused-ignores \
+		$(CURDIR)/src/toil/cwl/cwltoil.py
```

Comment:
@mr-c What's going on here? I figured this was a mistake since cwltoil.py is already not ignored in the main `mypy-with-ignore.py`, but now I'm finding that `mypy` on an individual file can give different results than mypy on a collection of files, and I suspect this might be doing *something* but can't figure out what.

Reply:
I want to get rid of this since I think it makes `mypy` scan most of the code twice, so `make mypy`, which we need to run all the time, gets about twice as slow.